internal/dataplane/sendconfig/sendconfig.go

```diff
@@ -94,9 +94,9 @@ func PerformUpdate(
 	promMetrics.RecordPushSuccess(metricsProtocol, duration, client.BaseRootURL())
 
 	if client.IsKonnect() {
-		logger.V(util.InfoLevel).Info("successfully synced configuration to Konnect")
+		logger.V(util.InfoLevel).Info("Successfully synced configuration to Konnect")
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/dataplane/sendconfig/sendconfig.go#L65
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/dataplane/sendconfig/sendconfig.go#L67
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/dataplane/sendconfig/sendconfig.go#L131
These are not changed.